care/facility/api/viewsets/patient.py

```diff
@@ -1,14 +1,18 @@
+import datetime
 import json
 from json import JSONDecodeError
 
+from django.contrib.postgres.search import TrigramSimilarity
 from django.db.models.query_utils import Q
 from django_filters import rest_framework as filters
 from dry_rest_permissions.generics import DRYPermissionFiltersBase, DRYPermissions
 from rest_framework import serializers, viewsets
 from rest_framework.generics import get_object_or_404
+from rest_framework.mixins import ListModelMixin
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.mixins'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5344132)